src/scripts/main.js

```diff
@@ -3,7 +3,7 @@
 const people = [
   {
     name: 'Carolus Haverbeke',
-    sex: 'm',
+    sex: 'Male',
```

Comment:
Don't need to change this file, you need to move this logic to the your function, fix it everywhere
```suggestion
    sex: 'm',
```